src/rebar_file_utils.erl

```diff
@@ -120,18 +120,13 @@ mv(Source, Dest) ->
                                       [{use_stdout, false}, abort_on_error]),
             ok;
         {win32, _} ->
-            {ok, R} = rebar_utils:sh(
-                        ?FMT("move /y \"~s\" \"~s\" 1> nul",
-                             [filename:nativename(Source),
-                              filename:nativename(Dest)]),
-                        [{use_stdout, false}, return_on_error]),
-            case R of
-                [] ->
-                    ok;
-                _ ->
-                    {error, lists:flatten(
-                              io_lib:format("Failed to move ~s to ~s~n",
-                                            [Source, Dest]))}
+            case file:rename(Source, Dest) of
```

Comment:
Hm, I didn't know about this. I've tried to fix dependency downloading. Rebar downloads them to temporary directory and then moves to the target location. In original implementation of `rebar_file_utils:mv/2` Windows `copy` command complains that `file name is too long` or something like that (I saw russian localization of this message). So maybe, instead of patching this function, I should change corresponding logic in `rebar_fetch`, which, I believe, not intended to move directories across drives?